cmd/openshift-install/imagebased.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"context"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func newImageBasedCmd(ctx context.Context) *cobra.Command {
+	imagebasedCmd := &cobra.Command{
+		Use:   "imagebased",
```

Comment:
all other `openshift-install` commands use hyphens for spaces, so this should be `image-based`

Reply:
This was caused by my ignorance, updated as suggested.